test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -902,11 +902,21 @@ def test_slice_dynamic_script(self):
         class DynamicSliceModel(torch.jit.ScriptModule):
             @torch.jit.script_method
             def forward(self, x):
-                return x[1:x.size(0)]
+                return x[1:x.size(1)]
 
         x = torch.rand(1, 2)
         self.run_test(DynamicSliceModel(), x)
 
+    @skipIfUnsupportedMinOpsetVersion(10)
```

Comment:
I know you have enables this for scripting. But if this test is already passing tracing as well, could you please write it as a nn.Module and use enableScriptTest decorator?
I think going forward we can use this approach for all tests. We want to make sure both code-paths are supported, unless there's a known issue.